kbase-extension/static/kbase/js/widgets/appWidgets2/validators/sequence.js

```diff
@@ -1,77 +1,62 @@
-define(['bluebird', './resolver', './constants'], (Promise, ValidationResolver, Constants) => {
+define([
+    'bluebird',
+    'widgets/appWidgets2/validators/resolver',
+    'widgets/appWidgets2/validators/constants',
+], (Promise, ValidationResolver, Constants) => {
     'use strict';
 
-    function applyConstraints(value, constraints) {
-        if (value === null || value.length === 0) {
-            if (constraints.required) {
```

Comment:
This wasn't getting called because the assumed data structure of `constraints` is wrong. Moved this code out to the main function, since it's not particularly complex and there was another bit of code there that fixed the `if (constraints.required)` not working part.